### ISSUE TITLE
Add Domain/Account Assignment at Instance Creation

### ISSUE THREAD

**Issue by scottsignal** [apache/cloudstack#9612]
##### ISSUE TYPE

 * Enhancement Request

##### COMPONENT NAME

~~~
UI
~~~

##### CLOUDSTACK VERSION

~~~
4.19.1.1
~~~

##### CONFIGURATION

~~~
N/A
~~~

##### OS / ENVIRONMENT

~~~
KVM
~~~

##### SUMMARY

Add Account/Domain Assignment at instance creation.  At this time, you need to go through the instance creation wizard and then assign the account later on.

##### STEPS TO REPRODUCE

~~~
N/A - Workflow Enhancement
~~~

##### EXPECTED RESULTS

~~~
When creating an instance being able to assign Domain/Account out of the gate.  Proposed to add these as dropdowns under #8 Details Section in the wizard.
~~~

##### ACTUAL RESULTS

~~~
Not possible today unless done via API.  You need to create the instance and then circle back to assign domain/account.
~~~

**Comment by scottsignal** [comment]
@BryanMLima Yes, this will work.  I didn't think to search for `VM` since they are called `Instances` now.  Thanks for the response.

**Comment by DaanHoogland** [comment]
fixed in #8919
